### ISSUE TITLE
java.lang.NullPointerException: Attempt to get length of null array  (BLECentralPlugin.java:486)

### ISSUE THREAD

**Issue by markterrill** [don/cordova-plugin-ble-central#504]
> java.lang.NullPointerException: Attempt to get length of null array 12-12 13:26:21.367 28154 28154 E AndroidRuntime: 	at com.megster.cordova.ble.central.BLECentralPlugin.findLowEnergyDevices(BLECentralPlugin.java:486)

Offending line:
```
if (serviceUUIDs.length > 0) {
```

Suggested fix:
```
if (serviceUUIDs != null && serviceUUIDs.length > 0) {
            bluetoothAdapter.startLeScan(serviceUUIDs, this);
        } else {
            bluetoothAdapter.startLeScan(this);
        }
```

Apologies, I'm a little time poor this week but hope this helps

**Comment by markterrill** [comment]
PR submitted

**Comment by don** [comment]
Fixed by #505
